magic-methods/code_stego/code_stego/decoder.py

```diff
@@ -0,0 +1,46 @@
+"""Implements decoder function"""
+
+import numpy as np
+import PIL.Image
+
+from .constants import (
+    HEADER_SIZE, MAGIC_NUMBER, USED_BITS_MASK, USED_BITS_PER_BYTE
+)
+
+
+def decode(image: PIL.Image.Image):
+    """Extracts the text from encoded image"""
```

Comment:
This function could use a bit more comprehensive doc compared to how the encoder function was documented such as mentioning the algorithm used while encoding the image with code.